changelog.md

```diff
@@ -0,0 +1,45 @@
+# Changelog
+All notable changes to this project will be documented in this file.  
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## Unreleased
+##  - 2022 - August - 
```

Comment:
Could you please explain why this line is obsolete ?

Reply:
@MariaMaximova, because it is `Unreleased` section. It is not related to date: you can add new feature to `dev` at the end of August and merge new feature later in September or in October. Thus, this line should be changed. That's why I'm asking if is obsolete and you can remove it.

Reply:
Please check file [Readme.md](https://github.com/MariaMaximova/convert_temp/pull/9/commits/acf053fd5a80cfe6f667f6b1463f29e0bee73d61#diff-1550ec65ac92f65817fc28928dfef526912b5f52356ff43651369bae92f56031)